reconcile/openshift_resources.py

```diff
@@ -267,8 +350,7 @@ def run(dry_run=False):
                         "Skipping resource '{}/{}' in '{}/{}'. "
                         "Present w/o annotations."
                     ).format(resource_type, name, cluster, namespace)
-                    logging.error(e_msg)
-                    errors.append(e_msg)
+                    error(e_msg)
```

Comment:
I think this error would display the last error_prefix that was updated inside fetch_data

Reply:
fixed by resetting the prefix